src/controllers/directorDateOfBirth.controller.js

```diff
@@ -14,6 +14,11 @@ const {
 } = require('../dynamics')
 
 module.exports = class DirectorDateOfBirthController extends BaseController {
+  checkOnlyDigits (number) {
+    const regex = /^\d*$/
+    return !!regex.exec(number)
+  }
+
```

Comment:
This needs test coverage

Reply:
Added test for this, plus similar tests for the permit holder DOB